bigquery/samples/tests/conftest.py

```diff
@@ -78,6 +87,13 @@ def table_id(client, dataset_id):
     client.delete_table(table, not_found_ok=True)
 
 
+@pytest.fixture
+def table_w_data(client):
```

Comment:
Let's call this `table_with_data`

Reply:
I changed this fixture to return `table_id` instead of table reference. That's why I assume better name is gonna be `table_with_data_id`.